api/v1/aggregate/aggregate.proto

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Hubble
+
+syntax = "proto3";
+
+package aggregate;
+
+option go_package = "github.com/cilium/cilium/api/v1/aggregate";
+
+// Aggregate contains flow aggregation counters
```

Comment:
Why do we create a new protobuf definition here? What speaks against adding this to the existing packages?